README.md

```diff
@@ -1,109 +1,48 @@
 # Node challenge: Pet Adoption RESTful API
-Este repositorio pretende ser una plantilla rápida para ayudar a practicar conceptos sobre APIs con NodeJS.
+## Overview
+This repository is intended to be a quick template to help practice API concepts with NodeJS.
 
-## Requerimientos funcionales
-Un Refugio de Animales local que quiere crear un sitio web para facilitar la adopción de mascotas. El proyecto en general busca crear una plataforma que conecte a los posibles adoptantes con las mascotas disponibles para adopción.
-La plataforma debería permitir a los usuarios administradores ver, gestionar perfiles de mascotas, revisar solicitudes de adopción abiertas y aprobar/rechazarlas, mientras que en el lado publico del API debe permitir a los usuarios ver las mascodas disponibles para adoptar, así como enviar solicitudes de adopción.
+A local Animal Shelter that wants to create a website to facilitate pet adoption. The overall project seeks to create a platform that connects potential adopters with pets available for adoption.
```

Comment:
gramatica: A local animal shelter wants... (eliminar "that" y animal shelter en minusculas)